doc/building_macos.md

```diff
@@ -0,0 +1,102 @@
+# Building Julius for Mac OS
+
+## Installing the dependencies
+
+In order to be able to build Julius, you need to install the dependencies first.
+
+The dependencies are:
+
+* The command line developer tools;
+* Homebrew;
+* CMake;
+* The `SDL` library;
+* The `SDL_mixer` library.
+
+If you already have all of them installed, you can proceed to [build Julius](#building-julius).
+
+
+### Installing the command line developer tools
+
+To install the command line developer tools, open a terminal window and type:
+
+    $ xcode-select --install
+
+The following window should appear:
+
+![](images/building_macos_1.png)
+
+Click `Install` and then accept the license agreement.
+
+The developer tools will download and install.
+
+![](images/building_macos_2.png)
+
+
+### Installing the remaining dependencies
+
+In order to install the remaining dependencies (CMake, SDL and SDL_mixer), you must first install [Homebrew](https://brew.sh).
+
+Homebrew is a package manager for Mac OS (and also for Linux) which makes it easier to download and install many components such as libraries and developer tools.
+
+In order to install Homebrew, open a Terminal window and type the following:
+
+    $ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+
+Homebrew will then install. Note that it will take some time.
+
+After Homebrew is installed, it's time to install the remaining dependencies. To do so, type:
+
+    $ brew install cmake sdl2 sdl2_mixer
```

Comment:
Personally I'm using the official Framework variants for SDL2 and SDL_mixer, and only use homebrew for cmake. One can also use the GUI for cmake (https://cmake.org/download/). Perhaps two sections? One for homebrew, one for SDL as Frameworks and CMake using GUI?

Reply:
You're right, but my mac os install in Virtualbox kept crashing so I left the mac instructions to a minimum.

Reply:
@bvschaik I remembered why I didn't provide the instructions for a framework! That was my original idea and I even created the instructions, but on Mac OS Catalina Julius will not run at all because, for some reason, the `SDL2_mixer` framework is not signed. Using the one from `brew` worked, so I changed the instructions.

Reply:
I haven't upgraded my Mac yet (and will not do so in the near future: Lightroom 6 will not work on Catalina...), but I wonder: does the DMG from the Travis build that's in bintray work?

Reply:
Good question, I'll check it out tomorrow.